tests/amr/multiphysics_integrator/test_main.cpp

```diff
@@ -223,35 +253,60 @@ PHARE::initializer::PHAREDict<1> createIonsDict()
         = std::string{"MaxwellianParticleInitializer"};
     dict["ions"]["pop0"]["ParticleInitializer"]["density"] = static_cast<ScalarFunctionT>(density);
 
-    dict["ions"]["pop0"]["ParticleInitializer"]["bulkVelocity"]
-        = static_cast<VectorFunctionT>(bulkVelocity);
+    dict["ions"]["pop0"]["ParticleInitializer"]["bulk_velocity_x"]
```

Comment:
it might be too much work, but it's typical to use constexpr types for strings like this to avoid typos
e.g.
```
constexpr char* IONS_STR ="ions";
```